src/person.rb

```diff
@@ -19,6 +22,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
+  end
+
```

Comment:
- [ ] Same thing here, I will advise you make it explicit because this is expected to hold book and date as parameter.